### ISSUE TITLE
No warning for duplicate fields in JSON file

### ISSUE THREAD

**Issue by Remi-Gau** [bids-standard/legacy-validator#571]
Not sure this is normal but the validator seems OK with JSON files having several times the same field with different values attached to it.
For example I duplicated and then slightly changed the SliceTiming in a *bold.json file and this was not picked up by the validator.
I might be wrong but I suspect that this might be an issue for some apps or for future development of the BIDS ecosystem.

**Comment by chrisgorgo** [comment]
Yes that would be problematic. How did you end up with duplicate entries? By editing the file manually?

**Comment by Remi-Gau** [comment]
Sort of yeah. Started using the json files created by dicm2nii matlab function and then to be thorough I added manually the fields recommended in the BIDS spec. I had apprently missed some but the validator did not mention it but it is only when reading through the data set with spm_BIDS.m that I got a warning for duplicate fields.
I suspect this will be a rare case but throwing a warning about it could be a good thing.
BTW: pybids seems "blind" to this problem too. With 2 different "RepetitionTime", it seemed to take the second one (the latest in the json file) as valid. Worth raising an issue on the github repo for pybids?
